src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -10,7 +10,7 @@
     public User add(User user) {
         user.setId(++index);
         Storage.people.add(user);
```

Comment:
Direct manipulation of the Storage.people list without any form of concurrency control can lead to issues in a multi-threaded environment. Consider using thread-safe collections or adding synchronization mechanisms.

Reply:
do not change it